lib/Hakyll/Web/Template/Internal.hs

```diff
@@ -201,3 +203,12 @@ unsafeReadTemplateFile file = do
     tpl <- unsafeCompiler $ readFile file
     pure $ template $ readTemplateElemsFile file tpl
 
+
+--------------------------------------------------------------------------------
+unsafeReadTemplateFile' :: FilePath -> String -> Compiler Template
+unsafeReadTemplateFile' file defValue = do
+    tpl <- unsafeCompiler $ do
+      isLocal <- doesFileExist file
+      if isLocal then readFile file
+      else return defValue
+    pure $ template $ readTemplateElemsFile file tpl
```

Comment:
I would put this function in the `where` clause of `renderFeed`, since we're not really using it anywhere else, and the name isn't really general enough to be true general-purpose.